src/controllers/users.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const {
+  getAll,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/usersServer');
+
+function getAllAction(req, res) {
+  const users = getAll();
+
+  res.send(users);
+};
+
+function addUsersAction(req, res) {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(422);
```

Comment:
Send status 400 instead of 422 when required parameter is not passed

Reply:
done